span-normalizer/span-normalizer-api/src/main/avro/trace-identity/IpIdentity.avdl

```diff
@@ -0,0 +1,10 @@
+@namespace("org.hypertrace.core.spannormalizer")
+protocol IpIdentityProtocol {
+  record IpIdentity {
+    string tenant_id;
```

Comment:
can we make union{string, null} tenant_id = null; for all. in this way it will be easy to remove few of the attributes in future if needed